docs/run-platform-server/configuring.mdx

```diff
@@ -0,0 +1,145 @@
+
+- If you want to connect Horizon to a different stellar network or override any of the defaults that horizon will make, the key configurations that can be set are: `--history-archive-urls`, `--captive-core-config-path`, `--network-pass-phrase`, `--captive-core-storage-path`, `--stellar-core-url`.
+
+- `--db-url`, required, specifies the Horizon database. It's value follows this format: `dbname=<pg_dbname> user=<pg_user_name> password=<pg_user_pwd> host=<host_address>`
+
+- `--log-level`, optional, can be one of 'info', 'error', 'debug'.
+
+- Horizon will create a sub-directory under the current working directory of the o/s process to store captive core runtime data files. Expect up to 10GB usage(**TODO - verify the max storage needs of core/buckets on disk?**). You can override this location with the optional `--captive-core-storage-path`, set to a directory on file system which captive core wiull store the runtime files.
+
+### Passing configurations to Horizon
+
+Depends on how Horizon was installed:
+
+- bare-metal
+  - non-package manager, we recommend using environment variables to pass configurations. There are many tools you can use to manage them, such as [direnv](http://direnv.net/) or [dotenv](https://github.com/bkeepers/dotenv).
+  - [package manager](./installing.mdx#package-manager), the provided `stellar-horizon-cmd` wrapper will import a configuration from `/etc/default/stellar-horizon`, hence, if you want to set configurations, edit the file at `/etc/default/stellar-horizon`.
+    <Alert>
+      This script invokes Horizon with the `stellar` user, so make sure that
+      permissions for the user are set up accordingly. The current working
+      directory should be writable for this user and the user should be able to
+      execute the `horizon` and `stellar-core` binaries; etc.
+    </Alert>
+- containerized
+  - non-Helm, pass all configuration parameters to the horizon docker image as [docker environment variables](https://docs.docker.com/engine/reference/commandline/run/#env).
+  - Helm, pass all configuration parameters in the [Helm Install command](https://helm.sh/docs/helm/helm_install/)
+
+## Prepare the Database first time
+
+Before running the Horizon server first time, you must initialize the Horizon database. This database will be used for all of the information produced by Horizon, most notably historical information about transactions that have occurred on the Stellar network.
+
+To prepare a database for Horizon's use, first ensure it is blank. It's easiest to create a new database on your PostgreSQL server specifically for Horizon's use. We recommend creating a new user(role) in postgres dedicated for horizon's db and assigning that user(role) as the owner of the new horizon db. To illustrate this as example using `psql`, first login to databse server using the `psql` cli as a superuser, and then create the new user(role) and db for horizon:
+
+```
+postgres=#
```

Comment:
When I was setting up PostgreSQL locally I found the `createuser` and `createdb` utilities that come bundled with the installation of Postgres easy to use. 
1. Create a new user called 'horizon' with a password
`createuser -P horizon`  
2. Create a new database called 'horizon' with the user 'horizon' as the owner
 `createdb -O horizon horizon`  

Reply:
Thanks for providing the reasoning. It makes sense and I agree with you in that we should opt for the lowest common denominator.